cmd/mysql/delete.go

```diff
@@ -44,22 +44,41 @@ var deleteEverythingCmd = &cobra.Command{
 	Run:       runDeleteEverything,
 }
 
+type DeleteHandler struct {
+	*internal.DeleteHandler
+	permanentObjects map[string]bool
+}
+
 func runDeleteEverything(cmd *cobra.Command, args []string) {
-	deleteHandler, err := newMySqlDeleteHandler()
+	deleteHandler, err := NewMySqlDeleteHandler()
 	tracelog.ErrorLogger.FatalOnError(err)
 
+	if p := deleteHandler.permanentObjects; len(p) > 0 {
+		tracelog.InfoLogger.Printf("found permanent objects %s\n", func() string {
```

Comment:
Maybe consider converting to slice and then strings.Join(obj, ",")?

Reply:
ok